doc/apimanager-reference.md

```diff
@@ -7,54 +7,56 @@ One APIManager custom resource per project is allowed.
 ## Table of Contents
 
 * [APIManager](#apimanager)
-   * [APIManagerSpec](#apimanagerspec)
```

Comment:
This change is generated with the tool? If so, I am ok with it. Otherwise, we should stick to the tool output to be more easy to manage.

Reply:
It was generated bye the tool yes